front-end/src/Components/Snack.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+import { Link } from 'react-router-dom';
+import regularHeart from '../assets/heart-regular.png';
+import solidHeart from '../assets/heart-solid.png';
+import noImage from '../assets/no-image.png';
+
+function Snack({ snack, index }) {
+	const imageDecider = snack.image_url ? (
+		<img
+			src={snack.image_url}
+			alt='snack'
+			width='100'
+			height='106'
+			className='snack-info-image'
+		/>
+	) : (
+		<img
+			src={noImage}
+			alt='default'
+			width='100'
+			height='106'
+			className='snack-info-image'
+		/>
+	);
+
+	return (
+		<Link to={`/snacks/${index}`}>
+			<div className='snack-card'>
+				<section>{imageDecider}</section>
+
+				{snack.is_healthy ? (
+					<span>
+						<img src={solidHeart} alt='solid heart' width='25' height='25' />
+
+						<p>{snack.name}</p>
+					</span>
+				) : (
+					<span>
+						<img
+							src={regularHeart}
```

Comment:
We could also use the Heart Health component here to determine what heart is rendered

Reply:
I agree. @JedeBrahma maybe we can attempt to write the code for this in the HeartHealth file?

Reply:
yes Michael on my end I did do it in the HeartHealth file - once i feel i am at a comfortable place with the fspp - we should make some time and do that - its pretty simple - just like adding the confirm health to backend.